src/lib/components/workflow/relationships/workflow-family-node-tree.svelte

```diff
@@ -207,6 +227,115 @@
     on:click={(e) => nodeClick(e, root)}
     on:keypress={(e) => nodeClick(e, root)}
   >
+    {#if root?.scheduleId}
+      <line
+        x1={x}
+        y1={y}
+        x2={x}
+        y2={y - 2.5 * radius}
+        stroke-dasharray="3 2"
+        class="stroke-slate-100 stroke-2 transition-all duration-300 ease-in-out dark:stroke-slate-800"
+      />
+      <line
+        x1={x - 5 * radius}
+        y1={y - 2.5 * radius}
+        x2={x}
+        y2={y - 2.5 * radius}
+        stroke-dasharray="3 2"
+        class="stroke-slate-100 stroke-2 transition-all duration-300 ease-in-out dark:stroke-slate-800"
+      />
+      <a href={routeForSchedule({ namespace, scheduleId: root.scheduleId })}>
+        <text
+          x={x - 5.25 * radius}
+          y={y - 2.5 * radius}
+          fill="currentColor"
+          text-decoration="underline"
+          text-anchor="end"
+          font-weight="500">{root.scheduleId}</text
```

Comment:
Looks like the schedule Id can go off the page ⬇️ 

![Screenshot 2025-04-18 at 5 19 32 PM](https://github.com/user-attachments/assets/e30d4ff6-b626-4046-b463-965299830d69)

Reply:
SVG text doesn't have widths/text-wrap. However you can zoom out or pan to see the full schedule name, so I think this is okay.